crates/biome_service/src/file_handlers/mod.rs

```diff
@@ -314,7 +316,10 @@ impl DocumentFileSource {
             DocumentFileSource::Css(_)
             | DocumentFileSource::Graphql(_)
             | DocumentFileSource::Json(_) => true,
+            #[cfg(not(feature = "experimental-html"))]
             DocumentFileSource::Html(_) => false,
+            #[cfg(feature = "experimental-html")]
+            DocumentFileSource::Html(_) => true,
```

Comment:
```suggestion
            DocumentFileSource::Html(_) => {
            	if cfg!(feature = "experimental-html") {
            		true
            	} else {
            		false
            	}            
            },
```

I think this should work too, but test it first